client/python/README.md

```diff
@@ -1,13 +1,61 @@
-# Armada Python client
+**Armada Python client
+-
 
 
-Python client wrapping the gRPC services defined in `submit.proto` and `events.proto`; allows for
+Getting Started:
+-
+On Linux Machine
+-
+1) Open your favorite shell, for example, good old [Bourne Again SHell, aka, bash](https://www.gnu.org/software/bash/), the somewhat newer [Z shell, aka, zsh](https://www.zsh.org/), or shiny new [fish](https://fishshell.com/).
 
-- submitting, cancelling, and reprioritising jobs, and
-- watching for job events.
+2) Install [Git](https://git-scm.com/) by running
+   sudo apt-get install git-all on [Debian-based](https://www.debian.org/)
+   distributions like [Ubuntu](https://ubuntu.com/), or
+   sudo dnf install git on [Fedora](https://getfedora.org/) and closely-related
+  [ RPM-Package-Manager](https://rpm.org/) based distributions like
+   [CentOS](https://www.centos.org/). For further information see
+   [Installing Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git).
+
+3) Install [poetry](https://python-poetry.org/) on your local machine by following the instructions on [python poetry documentation](https://python-poetry.org/docs/). 
+
+4) Fork the G-Research Armada repository to your personal repository.
+
+5) Clone the forked armada repository on your personal git repository by running git clone [Armada Repo](https://github.com/G-Research/armada.git) and navigate into the new directory Armada project by running cd armada.
+
+On Mac Machine
+-
+
+1) Open your favorite shell, for example, [Homebrew](https://brew.sh/) or [MacPorts](https://www.macports.org/).
+
+2) Install [Git](https://git-scm.com/) by running `$ brew install git` or `$ sudo port install git` or follow the instructions [here](https://www.atlassian.com/git/tutorials/install-git)
+
+3) Install [Python](https://docs.python-guide.org/starting/install3/osx/)
```

Comment:
I probably would suggest pyenv or something that allows more than one python version.  We also use python 3.9 in our CI so we should tell them to install python 3.9.  I don't think we support 3.10 yet.